### ISSUE TITLE
With the new conveyer belt sprites, the arrows sometimes face the wrong way 

### ISSUE THREAD

**Issue by NanoCats** [tgstation/tgstation#64274]
## Reproduction:

Observe this picture:
![image](https://user-images.githubusercontent.com/14065903/150292798-7d00a654-a42d-4aaf-a381-cf2de06adf05.png)

Now, if you place a conveyer belt after the addition of our new belt sprites, they indicate a direction. But the direction of the arrows sometimes does not correctly correlate to the direction the belt is actually moving in if you swap it out with a screw driver 

Observe the tile one down, one left of my player character in this image. It is one of the ones that facilitates moving crates around the cargo shuttle to the far side near the exit belt.  The issue is that the arrows face the wrong way, and the arrows showing which way the belt goes is like, the entire point?????

Furthermore, I can't solve this issue by wrenching, as there is no ←↓ configuration for the belt where the directional arrows face the correct way, no matter how many times you rotate the belt by wrenching. 

Ultimately, this doesn't "break" the game, but it undermines the point of the pretty new sprites for conveyer belts.

**Comment by NanoCats** [comment]
TL;DR the arrows should swap direction when you screwdriver the belt so that the arrows actually work to show you which way the belt goes.

**Comment by Bawhoppen** [comment]
Probably a dir issue from the resprite. If so, easy fix

**Comment by GoldenAlpharex** [comment]
@NanoCats So, this both is and isn't a bug.

There's something to do exactly what you're asking for, and it's done by right-clicking the belt with a screwdriver. It inverts both the arrow and the direction of the belt.

Now, the bug is that there wasn't a call to update the appearance of the belt when that was done, which meant that you had to activate the belt to notice it, so I don't blame you for thinking it was broken. I'm fixing that, should have a PR up for it shortly.
